core/src/main/java/org/apache/calcite/sql/SqlCall.java

```diff
@@ -191,7 +191,7 @@ public int operandCount() {
    * Returns a string describing the actual argument types of a call, e.g.
    * "SUBSTR(VARCHAR(12), NUMBER(3,2), INTEGER)".
    */
-  protected String getCallSignature(
+  public String getCallSignature(
```

Comment:
This change on method visibility, although minor, should be considered a breaking change and IMO mentioned briefly on the RN: any downstream project that had overridden it as `protected` would have a compilation issue with the new version, and would need to adjust their code to override it as `public`.

Reply:
what should I do about it?

Reply:
Usually in these cases, to make sure the Release Manager won't forget about it, what we do is editing the `history.md`: in that file, at the top, there is the "skeleton" with the sections of the next release (commented out), you could add a point in there in the "breaking changes" section mentioning this.

Reply:
I have added this to history.md. Thank you for pointing this out.